pylossless/dash/mne_visualizer.py

```diff
@@ -20,6 +20,12 @@ def _add_watermark_annot():
     from .css_defaults import WATERMARK_ANNOT
     return WATERMARK_ANNOT
 
+def _annot_in_timerange(annot, tmin, tmax):
+    annot_range = np.arange(annot["shape"]['x0'], annot["shape"]['x1'] + 1)
+    return any([tmin <= this_sec <= tmax
+                for this_sec
+                in annot_range])
+
```

Comment:
Great catch.....  You are right.  I think this matters for the following scenario:
the following shape SHOULD not be included in the viewable annotations:
```
tmin = 20 # time window min
tmax = 30 # time window max
shape_x0 = 10
shape_x1 = 19.01
```
My code `any([tmin <= this_sec <= tmax for this_sec in np.arange(shape_x0, shape_x1 +1)])` would return `True` because the comprehension includes `20 <= 20 < 30`.. When it shouldn't (`shape_x1` is `19.01` not `20`):
```
print(np.arange(shape_x0, shape_x1 +1))
# array([10., 11., 12., 13., 14., 15., 16., 17., 18., 19., 20.])
```

FWIW, our dashboard would "plot" that shape but because we fix the `xaxis` range it just wouldn't show up. So I don't think it would cause an error, BUT it's still a logical fallacy so I agree we should find a better solution.

I think your proposed solution would work... I'm testing it out to confirm.